sdk/src/signer/keypair.rs

```diff
@@ -16,11 +18,13 @@ use {
         io::{Read, Write},
         path::Path,
     },
-    wasm_bindgen::prelude::*,
 };
-
+/// wasm-bindgen version of the Keypair struct.
+/// This duplication is required until https://github.com/rustwasm/wasm-bindgen/issues/3671
+/// is fixed. This must not diverge from the regular non-wasm Keypair struct.
+///
```

Comment:
Sorry, maybe I'm missing something, but is this comment needed? There's only one version of Keypair because it doesn't require `#[wasm_bindgen(skip)]` anywhere

Reply:
ah woops